hc/accounts/tests/test_profile.py

```diff
@@ -18,8 +18,10 @@ def test_it_sends_set_password_link(self):
         self.alice.profile.refresh_from_db()
         token = self.alice.profile.token
         ### Assert that the token is set
-
+        self.assertEqual(self.alice.profile.token, token)
         ### Assert that the email was sent and check email content
+        self.assertEqual(len(mail.outbox),1)
+        assert "link" in mail.outbox[0].body
```

Comment:
Please use assertContains method available in Django SimpleTestCase class.
It provides metadata about the test on failure.

I'm well aware of other testing frameworks promoting use of assert stmts such as py.test but I think that if we are using Unittest here we should take advantage of the good things that it provides us with.
https://docs.python.org/2/reference/simple_stmts.html#the-assert-statement